README.md

```diff
@@ -1,2 +1,26 @@
 # as-chacha20poly1305
-chacha20poly1305 in AssemblyScript
+
+![ES Version](https://github.com/StableLib/stablelib/tree/master/packages/chacha20poly1305)
```

Comment:
Try to check all links in the repo

Reply:
I just want to refer to the original implementation, fixed it